src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -397,6 +397,34 @@ public static void Add(double realLeft, double imaginaryLeft, double realRight,
             VerifyRealImaginaryProperties(result, expectedReal, expectedImaginary);
         }
 
+        [Theory]
+        [MemberData(nameof(Add_TestData))]
+        [MemberData(nameof(Random_4_TestData))]
+        [MemberData(nameof(Invalid_4_TestData))]
+        public static void AddDouble(double realLeft, double imaginaryLeft, double realRight, double imaginaryRight)
```

Comment:
These aren't the key tests I would like to have for this code. Basically, these tests repeat the algorithms in the implementation and require the same result. The tests I would prefer to have would verify that the mixed arithmetic operators give the same result as the pure complex operators, e.g. z + a == z + ((Complex) a), etc.

Reply:
I agree. It would be useful to confirm that `double op complex` produces the same result as `((complex)(double)) op complex` for some of these tests